### PR TITLE
Update documentation to include better graph examples

### PR DESCRIPTION
**Pull request by tdensmore**
@jdewinne I split the single PR into two. this is the documentation pull request

### CODE REVIEW

File: content/reference/v1beta1/application.md [hunk 1]
```diff
@@ -51,98 +52,197 @@ spec:
...
+icon: "data:image/gif;base64,R0lGQBEAPeoAJ -- LOTS OF DATA TRIMMED FOR BREVITY -- z595kzAAOw=="
+```
+
+```yaml
+icon: "data:image/png;base64,iVBORw0KGgoAA -- LOTS OF DATA TRIMMED FOR BREVITY -- BU5ErkJggg=="
+```
+
+```yaml
+icon: "data:image/jpeg;base64,/9j/4RiDRXhp -- LOTS OF DATA TRIMMED FOR BREVITY -- oElFTkSuQmCC"
+```
```
[the remark this answers is not in the document]
Comment (tdensmore) [reply content/reference/v1beta1/application.md:91]: I was just improving what is already there... but I can add more. Discuss?

**Comment by AmberAlston** [comment]
@tdensmore following up on some older PRs. Is this moving forward or can it be closed?

**Comment by tdensmore** [comment]
I would love for it to be aproved. @samantha-replicated can you help with eyeballs and approval?

**Comment by jonquil2002** [comment]
@tdensmore Hey, I'm closing this PR because the content improvements on the new docs site have rendered merging this content not feasible. But I know you put a lot of work into this and I'd like to pair with you to get anything that is still relevant over to the new site. PTAL the new docs site and let me know: https://docs.replicated.com/reference/custom-resource-application

Thanks!

**Pull request closed without merging**